Yank/multistate/multistatesampler.py

```diff
@@ -1237,11 +1242,26 @@ def _minimize_replica(self, replica_id, tolerance, max_iterations):
             replica_id + 1, self.n_replicas, initial_energy))
 
         # Minimize energy.
-        openmm.LocalEnergyMinimizer.minimize(context, tolerance, max_iterations)
+        if (max_iterations is None) or (max_iterations == 0):
+            fire_iterations = 1000
```

Comment:
Can you update the docstring of `MultiStateSampler.minimize()` describing this behavior? If I recall correctly, the current docs don't allow `None` as a valid value for `max_iterations`.